pkg/docker/config/config.go

```diff
@@ -76,17 +77,25 @@ func SetAuthentication(sys *types.SystemContext, registry, username, password st
 			})
 		// External helpers.
 		default:
-			err = setAuthToCredHelper(helper, registry, username, password)
+			path = fmt.Sprintf("docker-credential-%s", helper)
```

Comment:
@mtrmac Tried fixing this in lastest commit, I have a suggestion a regular path for file and `helper:%s` in case helper is used.Caller doesn't have a strict contract as of now its used here https://github.com/containers/common/pull/607 but in any case if calle want to check if it was a helper he can split for `helper:`. WDYT ?

Reply:
@mtrmac Added prefix `credential helper: %s` as suggested by @vrothberg.

Reply:
> but in any case if calle want to check if it was a helper he can split for `helper:`. WDYT ?

If we make such a promise, we can’t add any new storage mechanisms (and this would have better been a Go type, not a string for a caller to parse _if_ we wanted to make such a promise).

Note to self: Eventually change this to make no format commitments. The text content is not an API (and doesn’t match the current code right now).

Reply:
> Note to self: Eventually change this to make no format commitments. 

Filed #1256 .